x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/single_metric_config.ts

```diff
@@ -93,35 +95,52 @@ export function getSyntheticsSingleMetricConfig({ dataView }: ConfigProps): Seri
       },
       {
         id: 'monitor_errors',
-        field: 'state.id',
         label: i18n.translate('xpack.observability.expView.errors', {
           defaultMessage: 'Errors',
         }),
         metricStateOptions: {
           titlePosition: 'bottom',
           colorMode: 'Labels',
-          palette: {
-            name: 'custom',
-            type: 'palette',
-            params: {
-              steps: 3,
-              name: 'custom',
-              reverse: false,
-              rangeType: 'number',
-              rangeMin: 0,
-              progression: 'fixed',
-              stops: [{ color: '#E7664C', stop: 100 }],
-              colorStops: [{ color: '#E7664C', stop: 0 }],
-              continuity: 'above',
-              maxSteps: 5,
-            },
-          },
+          palette: getColorPallet('danger'),
         },
         columnType: FORMULA_COLUMN,
         formula: 'unique_count(state.id, kql=\'monitor.status: "down"\')',
         format: 'number',
       },
+      {
+        id: 'monitor_failed_tests',
+        label: i18n.translate('xpack.observability.expView.failedTests', {
+          defaultMessage: 'Failed tests',
+        }),
+        metricStateOptions: {
+          titlePosition: 'bottom',
+        },
+        columnType: FORMULA_COLUMN,
+        formula: 'count(state.id, kql=\'monitor.status: "down"\')',
```

Comment:
count on a field is doing a value count aggregation (https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-valuecount-aggregation.html) - just making sure this is the right thing in your situation.